### PR TITLE
Rearrange subflow tab items

### PR DESCRIPTION
**Pull request by HiroyasuNishiyama**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Proposed changes

This PR rearranges location of some UI items (category setting and instance count
message) of SUBFLOW template properties tab according to discussion on https://github.com/node-red/node-red/pull/2196 .

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: packages/node_modules/@node-red/editor-client/src/js/ui/editor.js [hunk 1]
```diff
@@ -953,8 +953,70 @@ RED.editor = (function() {
...
+                if (n.type === subflowType) {
+                    userCount++;
+                }
+            });
+            $("#red-ui-editor-subflow-user-count")
+                .text(RED._("subflow.subflowInstances", {count:userCount})).show();
+        }        
+        
         $('<div class="form-row">'+
+            '<i class="fa fa-tag"> '+
```
Comment (knolleary) [inline packages/node_modules/@node-red/editor-client/src/js/ui/editor.js]: This tag doesn't have a closing `</i>` tag which breaks the appearance of this row:

Local_Node-RED


In fact, we don't have icons on this tab at all... please don't add them.

File: packages/node_modules/@node-red/editor-client/src/js/ui/editor.js [hunk 2]
```diff
@@ -972,6 +1034,11 @@ RED.editor = (function() {
         // If a node has icon property in defaults, the icon of the node cannot be modified. (e.g, ui_button node in dashboard)
         if ((!node._def.defaults || !node._def.defaults.hasOwnProperty("icon"))) {
             var iconRow = $('<div class="form-row"></div>').appendTo(dialogForm);
+            $("<i/>", {
```
Comment (knolleary) [inline packages/node_modules/@node-red/editor-client/src/js/ui/editor.js]: Please remove

File: packages/node_modules/@node-red/editor-client/src/js/ui/editor.js [hunk 3]
```diff
@@ -1001,7 +1068,7 @@ RED.editor = (function() {
             $('<div id="red-ui-editor-node-icon">').text(node.icon).appendTo(iconButton);
         }
 
-        $('<div class="form-row"><span data-i18n="editor.portLabels"></span></div>').appendTo(dialogForm);
+        $('<div class="form-row"><i class="fa fa-external-link"/> <span data-i18n="editor.portLabels"></span></div>').appendTo(dialogForm);
```
Comment (knolleary) [inline packages/node_modules/@node-red/editor-client/src/js/ui/editor.js]: Please remove the icon

File: packages/node_modules/@node-red/editor-client/src/js/ui/editor.js [hunk 4]
```diff
@@ -953,8 +953,70 @@ RED.editor = (function() {
 
         var i,row;
 
+        if (node.type === "subflow") {
+            var categoryRow = $("<div/>", {
+                class: "form-row"
+            }).appendTo(dialogForm);
+            $("<i/>", {
```
Comment (knolleary) [inline packages/node_modules/@node-red/editor-client/src/js/ui/editor.js]: Please remove the icon

**Comment by knolleary** [comment]
I'm going to make these changes myself as part of the merge. I'm trying to put together some proposed changes to the rest of the subflow UI PR, but want to get this in first.

**Comment by HiroyasuNishiyama** [comment]
Thank you for the fix. 
Committed fixes on proposed changes on SUBFLOW UI (https://github.com/node-red/node-red/pull/2196).

**Pull request merged**